spec/support/shared_examples/services/chunkers/all_chunkers.rb

```diff
@@ -1,18 +1,16 @@
 RSpec.shared_examples "all chunkers" do
-  # Requires
-  # - `subject` to be instance of a chunker
-  # - the following `let` variables to be defined
-  #   - :text
-  #   - :text_type
```

Comment:
Could this comment still be helpful?

Reply:
`:text` and `:text_type` aren't required, but it does make me think we should be testing chunking all document types.  I think we're only testing .md files currently.